Tests/Functional/Controller/FrontEndEditorControllerTest.php

```diff
@@ -71,6 +73,70 @@ public function indexActionForLoggedInUserDoesNotRenderTeaWithoutOwner(): void
         self::assertStringNotContainsString('Sencha', $html);
     }
 
+    #[Test]
+    public function indexActionHasHeadline(): void
+    {
+        $this->importCSVDataSet(__DIR__ . '/Fixtures/Database/FrontEndEditorController/TeaAssignedToLoggedInUser.csv');
+
+        $html = $this->getHtmlWithLoggedInUser();
+
+        $expected = LocalizationUtility::translate('plugin.frontEndEditor.index.heading', 'tea')
+            ?? throw new \RuntimeException('Could not fetch translation value.', 1762183380);
```

Comment:
I'd prefer to not use the `??` operator to trigger method calls.

In this particular case, I'd say we add an `assertIsString` so that PHPUnit will let us know if the translation cannot be fetched.